samples/apps-with-ingress/nginx.conf

```diff
@@ -0,0 +1,11 @@
+server {
+    listen 80;
+
+    location /A {
+        proxy_pass http://host.docker.internal:5005/;
```

Comment:
The nginx config doesn't support env variables so we hard code this for now (which is breaks if we change the execution mode). We'll need to make file transformations like this more first class for these scenarios.